[Max Autotune][B200] Skip carveout tests

jananisriram opened the pull request:
Summary: Skip sm `carveout` tests on B200, as carveout is currently unsupported.

Test Plan:
```
buck2 test 'fbcode//mode/opt' fbcode//caffe2/test/inductor:max_autotune -c fbcode.nvcc_arch=b200a -c fbcode.enable_gpu_sections=true -c fbcode.platform010_cuda_version=12.8 -c fbcode.re_gpu_tests=False -- test_honor_sm_carveout_with_triton_tma
```

Differential Revision: D83395610




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

njriasan commented on torch/testing/_internal/common_utils.py:
```diff
@@ -1405,6 +1405,24 @@ def printErrors(self) -> None:
+        str: The model of the NVIDIA GPU or empty str if not found.
+    """
+    try:
+        model = subprocess.check_output(
```
You should just be able to use the existing information that doesn't depend on subprocess. See the blackwell tests in test_max_autotune.py.

jananisriram committed:
[Max Autotune][B200] Skip carveout tests (#164435)

Summary:

Skip sm `carveout` tests on B200, as carveout is currently unsupported.

Test Plan: `test_max_autotune.py`

Reviewed By: NikhilAPatel

Differential Revision: D83395610
